egs/hkust/s5/local/hkust_prepare_dict.sh

```diff
@@ -31,7 +31,7 @@ cat $dict_dir/words.txt | grep -v '[a-zA-Z]' > $dict_dir/lexicon-ch/words-ch.txt
 ##### produce pronunciations for english 
 if [ ! -f $dict_dir/cmudict/cmudict.0.7a ]; then
   echo "--- Downloading CMU dictionary ..."
-  svn co https://svn.code.sf.net/p/cmusphinx/code/trunk/cmudict \
+  svn co -r 13068 https://svn.code.sf.net/p/cmusphinx/code/trunk/cmudict \
```

Comment:
@naxingyu are the latest results in hkust based on this version of the dictionary ? 
fisher_swbd results are most probably based on this dictionary as @naxingyu ran them recently.  ASpIRE results are based on an older version of the dictionary and I am pretty sure it is same case with fisher_english results. However I am merging this pull request as we will later update these older results.